src/components/widgets/sales-opportunities-funnel/sales-opportunities-funnel.directive.coffee

```diff
@@ -34,13 +34,17 @@ module.controller('WidgetSalesOpportunitiesFunnelCtrl', ($scope, $q, ChartFormat
   w.initContext = ->
     if $scope.isDataFound = angular.isDefined(w.content) && !_.isEmpty(w.content.opps_per_sales_stage) && hasOneOpportunity(w.content.opps_per_sales_stage)
 
-      $scope.statusOptions = _.compact _.map w.metadata.sales_stage_selection, (status) ->
+      defaultWidget = ImpacWidgetsSvc.getDefaultWidget(w.name);
```

Comment:
Logic of having a "default widget" and dashboard.status_selection being based on this defaultWidget is not ok: what if the widget is removed from the dashboard? The status_selection values should be stored just as it is stored in the widget's metadata:
  ```
    dashboard.status_selection = {
      values: [ 'Status1', 'Status2' ]
      reach: 'dashboard'
    }
  ```
This way, even if the first widget is deleted, any new widget would be defaulted to the stored status selection.